cmd/cert-checker/main_test.go

```diff
@@ -211,8 +212,6 @@ func TestCheckCert(t *testing.T) {
 		"Certificate has incorrect key usage extensions":                                                 1,
 		"Certificate has common name >64 characters long (65)":                                           1,
 		"Policy Authority isn't willing to issue for '*.foodnotbombs.mil': Wildcard names not supported": 1,
-		"commonName exceeding max length of 64":                                                          1,
-		"Certificate contains unknown extension (1.3.3.7)":                                               1,
```

Comment:
Are there no linters for an unknown extension or an overly long common name in zlint? It seems like we wouldn't want to remove these test cases.

Reply:
Overly long common name is something we already check for (see L213), unknown extensions is a weird one. It's not really required by anything, hence why there isn't a zlint check for it.

We could add our own check, but I think it'd basically just be `len(certificate.Extensions) > 1`.

Reply:
(or actually, we need more logic because we use the SCT extension... blergh)

Reply:
> Overly long common name is something we already check for (see L213)

:+1: 

> It's not really required by anything, hence why there isn't a zlint check for it.
> We could add our own check, but I think it'd basically just be len(certificate.Extensions) > 1.

I think checking that would be worthwhile. Couldn't it be more strict and check additionally that if len(certificate.Extensions) == 1 that extensions[0] == MustStaple?

Reply:
Ended up having to do something slightly more complicated but I think it works...